go.mod

```diff
@@ -33,11 +33,11 @@ require (
 	github.com/moby/buildkit v0.0.0-20181107081847-c3a857e3fca0
 	github.com/mtrmac/gpgme v0.1.2 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1
-	github.com/openshift/api v0.0.0-20200205133042-34f0ec8dab87
+	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
```

Comment:
This is unexpected change.

Reply:
I will try to resolve this, but the other 2 are replaced anyway. They should be harmless. Please let me know what you think. Thank you.

Reply:
From what I checked this is somehow a transitive dependency that's coming only when I change operator-registry, if I bump openshift deps this will work just fine picking newer version, which makes me worried a lot about that bump.